### PR TITLE
Adding Flamemaster product

### PR DESCRIPTION
**Pull request by amd-toolchain-support**
About FlameMaster: Flamemaster is an open source C++ program package for 0D
combustion and 1D laminar flame calculations.

### CODE REVIEW

File: var/spack/repos/builtin/packages/flamemaster/package.py
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/flamemaster/package.py]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/flamemaster/package.py
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/flamemaster/package.py]: Please replace `'Heinz Pitsch` with his GitHub account.  This list is used by automation that depends on the entries being accounts.

**Review by tldahlgren — COMMENTED** [review]
Have not tested but LGTM.  Please correct the maintainers entry.

File: var/spack/repos/builtin/packages/flamemaster/package.py
Comment (amd-toolchain-support) [reply var/spack/repos/builtin/packages/flamemaster/package.py]: We could not get the GitHub account for FlameMaster author 'Heinz Pitsch' so updated FlameMaster product description section.

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**
